boards/README.md

```diff
@@ -33,6 +33,7 @@ that Tock supports.
 | [LiteX on Digilent Arty A-7](litex/arty/README.md)                   | RISC-V RV32I    | LiteX+VexRiscV | custom     | custom         | No            |
 | [Verilated LiteX Simulation](litex/sim/README.md)                    | RISC-V RV32I    | LiteX+VexRiscv | custom     | custom         | No            |
 | [ESP32-C3-DevKitM-1](esp32-c3-devkitM-1/README.md)                   | RISC-V-ish RV32I| ESP32-C3       | custom     | custom         | No            |
+| [QEMU RISC-V 32 bit `virt` platform](qemu_rv32_virt/README.md)       | RISC-V RV32IMAC | QEMU           | custom     | custom         | Yes (duh)     |
```

Comment:
Meh, I thought that we were in a good enough state to just say "yeah, sure, QEMU works with this board", but looking that the [downstream distribution situation](https://repology.org/project/qemu/versions) we really are better off specifing a version here. I'm still unsure though, is this supposed to be a minimum-supported or tested version? We should make the expectations clear, the `virt` target introduced here worked without issues on v5.2.0 and still works on v7.0.0. The Makefile contains a "tested" version and prints that, but that does not necessarily correspond to the minimum supported one. @alistair23 what do you think?

Reply:
I don't think we should worry about specifying a version here.

As long as QEMU has RISC-V support (which it has for years) then the virt machine will work. In the future I expect the virt machine itself to be versioned, and we can then specify version there. But I don't think we need to say a minimum or maximum QEMU version

Reply:
Okay, cool, then I'll just leave the README like that.